crates/solidity/testing/snapshots/bindings_output/expressions/named_args/input.sol

```diff
@@ -0,0 +1,21 @@
+contract Test {
+    enum Severity {
+        INFO,
+        ERROR
+    }
+
+    event Log(
+        string name,
+        Severity level
+    );
+
+    error Failure(
+        Severity severity,
+        string cause
+    );
+
+    function test() public {
```

Comment:
nit: to make such tests more maintainable/diffs easier to review, I suggest:

- splitting them into two smaller ones `emit_named_args` and `revert_named_args`
- removing the unrelated `enum`, and only use simpler types (strings/ints), as the added defs/refs are noise here, since we are not testing enum binding.